zeitschrift-fur-medienwissenschaft.csl

```diff
@@ -20,493 +14,297 @@
     <category field="humanities"/>
     <issn>1869-1722</issn>
     <summary>Style for zfm. Zeitschrift für Medienwissenschaft. Citations in notes; no bibliography.</summary>
-    <updated>2012-09-27T22:06:38+00:00</updated>
+    <updated>2014-09-13T22:01:31+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
-  <!--
-================================================================================
-   TODO
-================================================================================
--->
-  <!--
-================================================================================
-   TERMS
-================================================================================
--->
+  <!-- HINWEISE
```

Comment:
Can you clarify, please? (Where should the "Hinweise" go? Should they be
given as xml comments?)

> On 2015-01-04T15:20:35+0100, Rintze M. Zelle said:
>      <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>

>    </info>
> -  <!--
> -================================================================================
> -   TODO
> -================================================================================
> --->
> -  <!--
> -================================================================================
> -   TERMS
> -================================================================================
> --->
> +  <!-- HINWEISE

> Using `<summary/>` is probably better, since some programs (like
> Mendeley, I think) show style summaries to users in their style
> browser, but ignore XML comments.

> --- Reply to this email directly or view it on GitHub:
> https://github.com/citation-style-language/styles/pull/1313/files#r22439516

--
Till A. Heilmann, Dr. phil.

Online-Texte http://www.tillheilmann.info/german.php#OnlineTexts

Universität Siegen
Philosophische Fakultät
Medienwissenschaftliches Seminar
Adolf-Reichwein-Straße 2, Raum AR-K 611
57076 Siegen
Telefon: +49 271 740-4791
Telefax: +49 271 740-2731
www.uni-siegen.de/phil/medienwissenschaft/
www.multimediale-systeme.de